tests/lib/rules/self-closing-comp.js

```diff
@@ -107,73 +107,84 @@ ruleTester.run('self-closing-comp', rule, {
   invalid: [
     {
       code: 'var contentContainer = <div className="content"></div>;',
+      output: 'var contentContainer = <div className="content"/>;',
```

Comment:
Great, and to be clear here is how I'm seeing the two rules interact:

With  `'react/jsx-space-before-closing': [2, 'never']`:
Initial: `<Foo></Foo>`
Fixed: `<Foo/>`


With  `'react/jsx-space-before-closing': 2`:
Initial: `<Foo></Foo>`
Fixed: `<Foo />`

I have now reverted f6cc36a and updated documentation for `jsx-space-before-closing` page.